modules/hcp-ts-load-balancer-svcs.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ts-load-balancer-svcs_{context}"]
+= Load balancer services for the hosted cluster are not available
+
+If a hosted control plane is not coming fully online because the load balancer services are not becoming available, check events, details, and the kccm pod.
+
+.Procedure
+
+* Look for events and details that are associated with the load balancer service within the hosted cluster.
+
+* By default, load balancers for the hosted cluster are handled by the kubevirt-cloud-controller-manager within the hosted control plane namespace. Ensure that the kccm pod is online and view its logs for errors or warnings. To identify the kccm pod in the hosted control plane namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <hosted-control-plane-namespace> -l app=cloud-controller-manager
```

Comment:
```suggestion
$ oc get pods -n <hosted_control_plane_namespace> -l app=cloud-controller-manager
```
Use underscores in 'user-replaced-values'
See: https://redhat-documentation.github.io/asciidoc-markup-conventions/